README.md

```diff
@@ -1,5 +1,9 @@
 # Talks'n Slides
 
 - Easterhegg 20
+
   - [Video](https://media.ccc.de/v/eh20-4-rosenpass-ein-vpn-zum-schutz-vor-quantencomputern)
   - [Slides](https://raw.githubusercontent.com/rosenpass/slides/pdf/2023-04-easterhegg/slides.pdf)
+
+- Presentation @ IAV
+  - [Slides](https://raw.githubusercontent.com/rosenpass/slides/pdf/2023-07-04-IAV/slides.pdf)
```

Comment:
Any reason for absolute URIs?

Reply:
They are more robust, i.E. when somebody copies or downloads the repo they still work from any context.

Reply:
I think that the exact opposite is true. A local file is much more reliable than an online service which may have a downtime, requires internet or may change its URL format.

Reply:
I am not picky here though. Feel free to merge :-)